### PR TITLE
Added channels, playlist, playlist items, no tests

### PR DESCRIPTION
**Pull request by DaveDeveloper**
added channels, playlist, playlist items, no tests 

works tested manually with my youtube accounts.

### CODE REVIEW

File: lib/yourub/meta_search.rb [hunk 1]
```diff
@@ -51,6 +51,19 @@ def get(video_id)
       Yourub::Result.format(request).first
     end
 
+    def getChannels(client,username)
+      request = Yourub::REST::Channels.list(self,{:part => 'snippet,statistics,contentDetails',:forUsername => username})
+	Yourub::Result.format(request)
+    end
```
Comment (edap) [inline lib/yourub/meta_search.rb]: add an empty line in between methods

**Comment by edap** [comment]
Hey @DaveDeveloper , looks good.
Could you please add the specs for the 3 new rest resources, you can find an example here:
`https://github.com/edap/yourub/blob/master/spec/yourub/rest/categories_spec.rb`
just make a request, save the result in the fixture folder and load it.

File: lib/yourub/meta_search.rb [hunk 2]
```diff
@@ -51,6 +51,19 @@ def get(video_id)
       Yourub::Result.format(request).first
     end
 
+    def getChannels(client,username)
```
Comment (edap) [inline lib/yourub/meta_search.rb]: client parameter is not used and can be safely removed

**Comment by DaveDeveloper** [comment]
I will work on it when get back on the related project. it should be soon.

**Comment by edap** [comment]
why did you opened a pull request if you even didn't test it locally?

**Pull request merged**